src/Packagist/WebBundle/Command/CompileStatsCommand.php

```diff
@@ -117,10 +117,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $ids[] = $row['id'];
         }
 
-        // add downloads from the last 7 days to the solr index
-        $solarium = $this->getContainer()->get('solarium.client');
-
-        if ($verbose) {
+         if ($verbose) {
```

Comment:
changing this indentation looks wrong to me

Reply:
@stof You are right, that was a mistake ! Fixed.